### PR TITLE
Added created InstructorHome and imported all the components we built on Friday

### PR DESCRIPTION
**Pull request by Rodgers31**
What I did:

- Created in InstructorHome component to house all of the other components we worked on Friday

Files i changed:

- InstrucotorHome

Why they were changed:

- To organize our code better and have a single component that houses and display all of the other components that have to do with an instructor 

[Loom Video] !(https://www.loom.com/share/6418dd91d2394233b626bc7d06bd1bfc)

### CODE REVIEW

**Comment by fernando817mm** [comment]
Joey (comment isn't going through): 
Looks good, important to note for an upcoming PR that another team member will be working on:
- The main file that is housing all instructor components here is called InstructorHome.js
- The main file that is housing all parent components is currently called index.js
------------------------------------------------------------------------------------------
We should stick with index.js to hold all the other components. Main branch already has that formatting. Either way, looks good!

File: src/components/pages/InstructorHome/instructorHome.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import InstructorSidebar from './InstructorSidebar';
+import InstructorCalender from './InstructorCalender';
+import InstructorProfile from './InstructorProfile';
+
+const instructorHome = () => {
+  return (
```
Comment (Jie-chelchel) [inline src/components/pages/InstructorHome/instructorHome.js:7]: Same question as I mentioned in Michael's PR.

**Review by Jie-chelchel — APPROVED** [review]

**Review by fernando817mm — APPROVED** [review]

**Review by bradlylewis — APPROVED** [review]
Looks good. It makes sense to make index.js the main container for the components.

**Comment by JoeyMBrown** [comment]
Instructor home got changed to index.js
Old index.js has been removed
Component name is no exported with a starting capital character

**Pull request merged**